plugins/opener/Cargo.toml

```diff
@@ -0,0 +1,64 @@
+[package]
+name = "tauri-plugin-opener"
+version = "1.0.0"
+edition = { workspace = true }
+authors = { workspace = true }
+license = { workspace = true }
+repository = { workspace = true }
+links = "tauri-plugin-opener"
+
+[package.metadata.docs.rs]
+rustc-args = ["--cfg", "docsrs"]
+rustdoc-args = ["--cfg", "docsrs"]
+
+# Platforms supported by the plugin
+# Support levels are "full", "partial", "none", "unknown"
+# Details of the support level are left to plugin maintainer
+[package.metadata.platforms]
+windows = { level = "full", notes = "" }
+linux = { level = "full", notes = "" }
+macos = { level = "full", notes = "" }
+android = { level = "partial", notes = "Only allows to open URLs via `open`" }
+ios = { level = "partial", notes = "Only allows to open URLs via `open`" }
+
+
+[build-dependencies]
+tauri-plugin = { workspace = true, features = ["build"] }
+schemars = { workspace = true }
+serde = { workspace = true }
+urlpattern = { workspace = true }
+regex = "1"
+
+[dependencies]
+serde = { workspace = true }
+serde_json = { workspace = true }
+tauri = { workspace = true }
+thiserror = { workspace = true }
+open = { version = "5", features = ["shellexecute-on-windows"] }
+urlpattern = { workspace = true }
+regex = "1"
```

Comment:
Personally I would love to see it not to use regex, at least making this optional

Reply:
I am not sure if there is a better alternative, same dependencies are used by http plugin but I definitely want to avoid using it here.

Reply:
decided to go with `glob`, we already depend on it in tauri and is light weight enough while providing a simple syntax for matching against ulrs for secuirty.